src/utils/format-date.js

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function formatDate (date, locales) {
+  const jsDate = new Date(date)
+  const options = { year: 'numeric', month: 'long', day: 'numeric' }
+  const intlDateTimeFormatter = new Intl.DateTimeFormat(locales, options)
+
+  return intlDateTimeFormatter.format(jsDate)
+}
```

Comment:
No change needed here, but for context I created a similar util function but then resolved it was just as simple if not clearer to just include the date format in context with the associated code, for example in `views/partials/breaches.js`:

```js
function createBreachRows (data) {
  const locale = getLocale()
  const shortDate = new Intl.DateTimeFormat(locale, { year: 'numeric', month: '2-digit', day: '2-digit', timeZone: 'UTC' })
  const shortList = new Intl.ListFormat(locale, { style: 'narrow' })
  const longDate = new Intl.DateTimeFormat(locale, { dateStyle: 'long', timeZone: 'UTC' })
  const longList = new Intl.ListFormat(locale, { style: 'long' })
  const breachRowsHTML = data.verifiedEmails.flatMap(account => { ......
```
Setting the above constants allowed me to reuse the same custom format for different dates.  But, if it's more helpful for us to have this util, it doesn't hurt and I'm all for it 👍

Reply:
I find having utils for formatting dates in one place helpful. Only having to change them in one place helps with being mindful of adding formats.